src/selectivity-multiple.js

```diff
@@ -439,10 +445,17 @@ var callSuper = Selectivity.inherits(MultipleSelectivity, {
 
     _renderSelectedItem: function(item) {
 
-        this.$searchInput.before(this.template('multipleSelectedItem', $.extend({
+        var options = $.extend({
             highlighted: (item.id === this._highlightedItemId),
             removable: !this.options.readOnly
-        }, item)));
+        }, item);
+
+        this.$searchInput.before(this.template('multipleSelectedItem', options));
+
+        var $select = this.$('select');
+        if ($select.length) {
+            $select.append(this.template('selectOptionCompliance', options));
```

Comment:
Ah yes, I overlooked this is actually called form _rerenderSelection() itself :)